src/util/fields.py

```diff
@@ -676,7 +676,7 @@ def convert_field_to_html(cr, model, field, skip_inherit=()):
 
     # Update ir.default
     if table_exists(cr, "ir_default"):
-        json_value_html = pg_text2html("json_value")
+        json_value_html = pg_text2html("json_value").replace("<p>", '"<p>').replace("</p>", '</p>"')
```

Comment:
We should extract the text here.
```suggestion
        json_value_html = pg_text2html("json_value #>> '{}'")p>", '</p>"')
```

Reply:
but column [``json_value``](https://github.com/odoo/odoo/blob/de22c88aa475be2ab8413c3fc4632db4a622e134/odoo/addons/base/models/ir_default.py#L26) is varchar

Reply:
`json_value::json` then

Reply:
```
=> select json_value from ir_default
+--------------+
| json_value   |
|--------------|
| "en_US"      |
| "optional"   |
| "normal"     |
| 7            |
| 16           |
| 33           |
| 27           |
| "2025-01-01" |
| false        |
+--------------+
```
```
=> select json_value::json #>> '{}' from ir_default
+------------+
| ?column?   |
|------------|
| en_US      |
| optional   |
| normal     |
| 7          |
| 16         |
| 33         |
| 27         |
| 2025-01-01 |
| false      |
+------------+
```

Reply:
yes it works thanks for your suggestion